app/models/host/discovered.rb

```diff
@@ -149,18 +149,14 @@ def reboot
     proxy_url = self.proxy_url
 
     if proxy_url[:type] == 'proxy'
-      status = ForemanDiscovery::ProxyOperations.new(:url => proxy_url[:url], :operation => 'reboot').parse_put_operation
+      ForemanDiscovery::ProxyOperations.new(:url => proxy_url[:url], :operation => 'reboot').parse_put_operation
     else
-      status = ::ProxyAPI::BMC.new(:url => "http://#{self.ip}:8443").power :action => "cycle"
+      ::ProxyAPI::BMC.new(:url => "http://#{self.ip}:8443").power :action => "cycle"
     end
-
-    msg = status ? 'successful' : 'failed'
-    logger.info "ForemanDiscovery: reboot result: #{msg}"
-    status
-    rescue => e
-      logger.info "ForemanDiscovery: reboot result: failed"
-      logger.warn e.backtrace.join('\n')
-      raise e
+  rescue => e
+    logger.info(errmsg = N_("Unable to reboot %{name}: %{msg}"))
```

Comment:
Right, plus I should be using parameter of the Exception constructor. I am not going to interpolate into logs, only the exception.